scripts/ci/build_static_x86_64.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+set -e
+
+CONFIG_OPT="--prefix=/opt/odp ${CONF}"
+
+cd "$(dirname "$0")"/../..
+./bootstrap
+echo "./configure $CONFIG_OPT"
+./configure $CONFIG_OPT
+
+make clean
+
+make -j $(nproc)
+
+make install
+
+# Build and run sysinfo with installed libs
+pushd ${HOME}
+
+${CC} ${CFLAGS} ${OLDPWD}/example/sysinfo/odp_sysinfo.c -static -o odp_sysinfo_inst_static `PKG_CONFIG_PATH=/opt/odp/lib/pkgconfig:${PKG_CONFIG_PATH} pkg-config --cflags --libs --static libodp-linux`
```

Comment:
I think it would give much benefit as example/sysinfo/odp_sysinfo.c would not be replaced by the same variable. Variable would add one line to the script, and used only on the next two lines. This is similar to odp_sysinfo_inst_dynamic in build.sh.

Reply:
Similar to `CONFIG_OPT`, it just helps maintainability.

Reply:
Well, agree to disagree, resolving this.